example/multi-package/entrypoint.sh

```diff
@@ -4,9 +4,8 @@
 
 # we need to add example/multi-package ar as included import path
 # without it protoc could not find the foo.proto & hello.proto
-gripmock --stub=example/multi-package/stub --imports=example/multi-package/ \
-  example/multi-package/bar/bar.proto \
-  example/multi-package/foo.proto example/multi-package/hello.proto > gripmock.log 2>&1 &
+gripmock --stub=example/multi-package/stub \
+  example/multi-package/ > gripmock.log 2>&1 &
```

Comment:
please explain why it changed into this. and help me understand why the new format works